coalib/parsing/StringProcessing/Core.py

```diff
@@ -328,6 +328,39 @@ def escape(string, escape_chars, escape_with="\\"):
     return string
 
 
+def convert_to_raw(string, exceptions=""):
+    """
+    Converts a string to its raw form, converting all backslash to double
+    backslash except when the backslash escapes a character given in
+    exceptions.
+
+    :param string:     The given string that needs to be converted
+    :param exceptions: A list of characters that if escaped with backslash
+                       should not be converted to double backslash.
+    :return:           Returns the corresponding raw string.
+    """
+    always_exceptions = ['\'', '\"', 'a', 'b', 'f', 'n', 'r', 't', 'v', '/']
```

Comment:
that would mean that the path `test\test` doesn't work as `\t` is an escape char in python, we shouldn't do that! It's inconsistent as we don't interpret `\t` as tab